### ISSUE TITLE
Mail method not working at the moment

### ISSUE THREAD

**Issue by rjcorwin** [jywarren/plots#80]
Trying to reset my password but not getting the email from Drupal. I also tried to trigger one by commenting on one of my notes. http://publiclaboratory.com/notes/rjstatic/1-20-2012/recap-thermal-flashlight-workshop-12182011#comment-465  @jywarren Did you get a mail notification for that?

No error in the Drupal log but ... I found a mailhandler error http://publiclaboratory.com/admin/reports/event/770458.   Probably unrelated but might be worth looking at.

**Comment by jywarren** [comment]
yes, i got that mail update

damn, i gotta set aside some time to debug that

On Wed, Feb 15, 2012 at 5:49 PM, R.J. Steinert <
reply@reply.github.com

**Comment by rjcorwin** [comment]
Hm, maybe it's an issue on my end.  Nothing in spam, I'll poke a little right now to see if I can figure it out.

**Comment by rjcorwin** [comment]
When user rjstatic-admin submits user "R.J. Steinert" contact form with the "Send me a copy" option checked, rjstatic-admin get an email but "R.J. Steinert" does not.

Both rjstatic-admin and user "R.J. Steinert" are subscribed to the Thread http://publiclaboratory.com/notes/rjstatic/1-20-2012/recap-thermal-flashlight-workshop-12182011#comment-468 but when either user tries to submit a comment, neither users get emails.  

@jywarren Did you get the contact form email that rjstatic-admin sent warren?  Did you also get all of the email notifications for the thermal flashlight workshop thread?

**Comment by rjcorwin** [comment]
Stop the press! Just found some of those notifications in my spam folder.  I was monitoring my spam folder but gmail was not updating it...  need to look through all of these and make sure everything is there.

**Comment by rjcorwin** [comment]
Ok, contact form stuff is working fine (despite the fact it ends up in my spam folder).  Issues with Threads still remain.

**Comment by jywarren** [comment]
hmm, how does this differ from our debugging of this issue last time? I can
ping a few Plotters to see if it's a generalized problem. We had switched
everyone from HTML Mail to just Mail, no?

I'm getting lost of updates all the time. I wonder what it depends on?

On Wed, Feb 15, 2012 at 8:20 PM, R.J. Steinert <
reply@reply.github.com

**Comment by rjcorwin** [comment]
So it's not the mail method (because the warren user is getting
notification emails and other users are getting contact form emails) and
the fact that rjstatic-admin and "R.J. Steinert" are both not getting
notification emails suggest that it's not a permission problem (unless
there is some perm that rjstatic-admin doesn't have).  It could still be a
problem on my end somehow...  When I get a chance I'll try turning
"Immediate sending" off so I can see notifications collect in the queue.
Since the warren account is the super user account it would be good to get
some other users tested.

On Thu, Feb 16, 2012 at 1:39 PM, Jeffrey Warren <
reply@reply.github.com

**Comment by rjcorwin** [comment]
It looks like this was a problem with GMail.  I noticed last night that mail from site@publiclaboratory.com was going into my spam folder.  The weird thing was that not all of it would show up when checking spam.  I could see some notifications from Wiki page updates but when I reset my password, the email would not show up in either spam or the inbox.  The only way I could see the email was by searching for it.   So... gmail was somehow putting my mail from site@publiclaboratory.com "nowwhere".  :-/  I've created a filter to prevent all mail from site@publiclaboratory.com from going to spam, maybe that will help...
